### PR TITLE
SpikeHD.Dorion version 4.0.1

### PR DESCRIPTION
**Pull request by headquarter8302**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133525)

### CODE REVIEW

**Comment by headquarter8302** [comment]
@microsoft-github-policy-service agree

File: manifests/s/SpikeHD/Dorion/4.0.1/SpikeHD.Dorion.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SpikeHD.Dorion
+PackageVersion: 4.0.1
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{01AFEB88-BA8F-48D2-A3DF-FCC0B70D8723}'
```
Comment (Exorcism0666) [inline manifests/s/SpikeHD/Dorion/4.0.1/SpikeHD.Dorion.installer.yaml:8]:
```suggestion
ProductCode: '{01AFEB88-BA8F-48D2-A3DF-FCC0B70D8723}'
InstallerSwitches:
  Silent: /quiet
  SilentWithProgress: /passive 
```

**Comment by Exorcism0666** [comment]
Log:

```
2024-01-10 10:12:24.897 [CORE] Started applying motw using IAttachmentExecute to C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\SpikeHD.Dorion.4.0.1\48d0fccc6093f916f077bae51c1e042acce0411187633c0586d87468de4858b1
2024-01-10 10:12:37.776 [CORE] Finished applying motw using IAttachmentExecute. Result: -2147467259 IAttachmentExecute::Save() result: -2147467259
2024-01-10 10:12:37.780 [FAIL] Installer failed security check. Url: https://github.com/SpikeHD/Dorion/releases/download/v4.0.1/Dorion_4.0.1_x64_en-US.msi Result: 0x80004005
2024-01-10 10:12:37.780 [CLI ] Terminating context: 0x8a15002d at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DownloadFlow.cpp:1e6
```

**Comment by headquarter8302** [comment]
I am unfamiliar with how the checks happen and how they work. Why did the security check fail?

**Comment by Exorcism0666** [comment]
The package didn't pass a Defender or similar security scan. This might be a false positive so we can rescan. (unfortunately I'm not a moderator so I don't know if the command will work for my part) @headquarter8302

**Comment by Exorcism0666** [comment]
yeah

**Comment by stephengillie** [comment]
> > The package didn't pass a Defender or similar security scan. This might be a false positive so we can rescan. (unfortunately I'm not a moderator so I don't know if the command will work for my part) @headquarter8302
> 
> I noticed that. While installing the package locally, defender blocked it from running. Do all MSIX packages need to be passed by defender?

Yes, every package receives at least one Defender scan, as well as other security scans. 

https://learn.microsoft.com/en-us/windows/package-manager/package/repository#validation-process

**Comment by SpikeHD** [comment]
Hey all!

I've submitted the portable version of Dorion (contains the same contents of the MSI, just all in one place) to Microsoft's Defender team. I assume that is basically the only thing one can do to get this PR any further?

**Comment by stephengillie** [comment]
Hi @SpikeHD,

False positives usually pass about a day later, so I'll scan this again tomorrow.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**